kbs/docker/coco-as-grpc/Dockerfile

```diff
@@ -2,18 +2,24 @@ FROM rust:latest as builder
 ARG ARCH=x86_64
 ARG HTTPS_CRYPTO=rustls
 ARG ALIYUN=false
+ARG PLUGINS="nebula-plugin"
```

Comment:
I suggest that we leave default plugins empty. Also, we can add some guides in docs to show how to enable nebula when building KBS image.

For this we might need to add a new doc about plugins [here](https://github.com/confidential-containers/trustee/tree/main/kbs/docs).